CONTRIBUTING.md

```diff
@@ -0,0 +1,355 @@
+- [License information →](LICENSE.md)
+- [Original repository →][repohome]
+- [Issues →][repoissues]
+- [Pull requests →][repoprs]
+- [Milestones →][repomilestones]
+- [Projects →][repoprojects]
+
+## Contributor License Agreement
+
+Per the [GitHub Terms of Service][gh-tos], be aware that by making a
+contribution to this project, you agree:
+
+* to license your contribution under the same terms as [this project's
+  license][osmit], and
+* that you have the right to license your contribution under those terms.
+
+See also: ["Does my project need an additional contributor agreement? Probably
+  not."][cla-needed]
+
+[gh-tos]:     https://help.github.com/articles/github-terms-of-service/#6-contributions-under-repository-license
+[osmit]:      #open-source-license
+[cla-needed]: https://opensource.guide/legal/#does-my-project-need-an-additional-contributor-agreement
+
+
+## Code of conduct
+
+Harrassment or rudeness of any kind will not be tolerated, period. For
+specifics, see the [CODE_OF_CONDUCT](CODE_OF_CONDUCT.md) file.
+
+## Asking questions and reporting issues
+
+### Asking questions
+
+Please check the [README](README.md) or existing [issues][repoissues] first.
+
+If you cannot find an answer to your question, please feel free to hop on our 
+[gitter][gitterurl] [![Gitter](https://badges.gitter.im/bats-core/bats-core.svg)](https://gitter.im/bats-core/bats-core?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge) or [via IRC (#bats on freenode)][ircurl].
+
+### Reporting issues
+
+Before reporting an issue, please use the search feature on the [issues
+page][repoissues] to see if an issue matching the one you've observed has already
+been filed.
+
+### Updating or filing a new issue
+
+#### Information to include
+
+Try to be as specific as possible about your environment and the problem you're
+observing. At a minimum, include:
+
+#### Installation issues
+
+1. State the version of bash you're using `bash --version`
+2. If you're installing through homebrew, run `brew doctor`, and attach the 
+output of `brew info bats-core`
+
+#### Bugs/usage issues
+
+- State the version of bash you're using `bash --version`
+- Command line steps or code snippets that reproduce the issue
+- Any apparently relevant information from the [Bash changelog][bash-changes]
+
+[bash-changes]: https://tiswww.case.edu/php/chet/bash/CHANGES
+
+Also consider using:
+
+- bash's `time` builtin to collect running times
+- a regression test to add to the suite
+- memory usage as reported by a tool such as
+  [memusg](https://gist.github.com/netj/526585)
+
+### On existing issues
+
+1. DO NOT add a +1 comment: Use the reactions provided instead
+2. DO add information if you're facing a similiar issue as someone else, but 
+within a different context (e.g. different steps needed to reproduce the issue 
+than previous stated, different version of bash or bats, different OS, etc.) 
+You can read on how to do that here: [Information to include][#information-to-include]
```

Comment:
The "Information to include" section is missing. Did you mean to add it?

Reply:
PR Updated :)